Editor/CourseValidation/IValidationHandler.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using Innoactive.Creator.Core;
+
+namespace Innoactive.CreatorEditor.CourseValidation
+{
+    /// <summary>
+    /// ValidationHandler handles the validation of course the context.
```

Comment:
This description is very odd 🤔
Sorry, I am kind of block now and I can not thing of anything better to suggest for now 😕

Reply:
`ValidationHandler validates data objects, e.g. steps or conditions, of a given course and reports whether the object causes issues or is faulty, e.g. empty fields or invalid values in a behavior.`